nest_elephant_tvb/translation/FatEndPoint.py

```diff
@@ -0,0 +1,196 @@
+from mpi4py import MPI
+import pathlib
+
+def make_connections(path_to_files_receive, path_to_files_send, logger_master):
+    '''
+    Fat end point, here: connections to nest and tvb.
```

Comment:
Function documentation is unclear without additional information

Reply:
```suggestion
    Rich End Point. Still a first draft, will be changed to a proper interface.
    This function establishes two MPI intercommunicators. One to NEST and one to TVB.
    :param path_to_files_receive: path to file, store information about receiving MPI connection
    :param path_to_files_send: path to file, store information about sending MPI connection
    :param logger_master: main logger for the connections
```